st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -126,7 +128,7 @@ def _handle_schedule_execution(self, liveaction_api):
             LOG.exception('Unable to execute action. Unexpected error encountered.')
             abort(http_client.INTERNAL_SERVER_ERROR, str(e))
 
-    def _schedule_execution(self, liveaction, user=None):
+    def _schedule_execution(self, liveaction, user=None, **kwargs):
```

Comment:
I'm using kwargs here as a replacement for pecan.request to use in methods like `_get_from_model_kwargs_for_request`. Fixing problems like this requires complete refactoring of st2api. It's not what I'm doing here.